pkg/local_object_storage/shard/put.go

```diff
@@ -15,7 +15,10 @@ type PutPrm struct {
 }
 
 // PutRes groups the resulting values of Put operation.
-type PutRes struct{}
+type PutRes struct {
+	metaErr bool
```

Comment:
non-critical suggestion: enum?

just afraid of undefined state (e.g. both `true`)